code/modules/reagents/reagent_containers/food/snacks/grown.dm

```diff
@@ -77,7 +77,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/harebell
 	name = "harebell"
-	desc = "\"I'll sweeten thy sad grave: thou shalt not lack the flower that's like thy face, pale primrose, nor the azured hare-bell, like thy veins; no, nor the leaf of eglantine, whom not to slander, out-sweeten�d not thy breath.\""
+	desc = "\"I'll sweeten thy sad grave: thou shalt not lack the flower that's like thy face, pale primrose, nor the azured hare-bell, like thy veins; no, nor the leaf of eglantine, whom not to slander, out-sweeten�d not thy breath.\""

```

Comment:
Atom does things like these, it also did it with the "Did you know?" PDA string but there's no visible effect in Dreammaker or in-game